### DISCUSSION TITLE
Cloudflare Turnstile

### DISCUSSION THREAD

**Discussion by jorgb90** [breezefront/community#266] [3. Integrations]
Integration with this extension which integrates Cloudflare Turnstile would be great. The default Magento Google Recaptcha integration is slow and full of errors.

https://github.com/Pixel-Open/magento-cloudflare-turnstile

**Comment by jorgb90** [comment]
@vovayatsyuk Tried doing it myself without any luck. Shouldn't be to difficult I think though, but perhaps I am missing something.

**Reply by IbrahimS2** [reply]
@jorgb90 You will have to create a module and map/integrate it with Breeze, there is no copy/paste solution, you can refer to existing integrations to have an idea, otherwise a developer will be needed to accomplish it for you.

**Reply by jorgb90** [reply]
@IbrahimS2 Yes I know. Tried to do it myself and I get the scripts / components integrated through a custom module with breeze_default.xml but don't get the extension to work, so I am missing something.

**Reply by IbrahimS2** [reply]
@jorgb90 Please share the custom module, I will have a quick look
